sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/secret_client.hpp

```diff
@@ -235,6 +235,35 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Secrets {
         std::string const& version,
         KeyvaultSecretProperties const& properties,
         Azure::Core::Context const& context = Azure::Core::Context()) const;
+
+    /**
+     * @brief Back up the specified secret.
+     * Requests that a backup of the specified secret be downloaded to the client.
+     * All versions of the secret will be downloaded.
+     * This operation requires the secrets/backup permission.
+     *
+     * @param name The name of the secret<span class="x x-first x-last">.</span>
+     * @param context The context for the operation can be used for request cancellation.
+     *
+     * @return The The backup blob containing the backed up secret.
+     */
+    Azure::Response<BackupSecretData> BackupSecret(
```

Comment:
Just checked the name from BackupKey and we use the suffix `Result`. 
```suggestion
    Azure::Response<BackupSecretResult> BackupSecret(
```

Reply:
i thought about it , and if it would be suffixed Response instead of data for one thing it's redundant , as the return of the function becomes Azure::**Response**<XYZ**Response**>
Second , if the Backup/Restore APIs are to be consistent one should take as input the output from the previous one . 
.Net does not bother with these structures they have the following definitions 
public virtual Response<**byte[]**> BackupSecret(string name, CancellationToken cancellationToken = default)
and 
public virtual Response<SecretProperties> RestoreSecretBackup**(byte[] backup**, CancellationToken cancellationToken = default)
so you can potentially do var x = Backup .... Restore(x)

Reply:
the suffix we use is `Result`.

The name is expected to be like `Respone<ModelResult>`

Reply:
OK, i'll update it